system/lib/libcxxabi/src/gxx_personality_stub.cpp

```diff
@@ -0,0 +1,20 @@
+#include <stdint.h>
+#include <stdlib.h>
+
+
+#include "cxa_exception.h"
+#include "cxa_handlers.h"
+#include "emscripten.h"
+
+
+
+// TODO: Remove this when Rust doesn't need it anymore.
+// https://github.com/rust-lang/rust/pull/97888
+extern "C" _LIBCXXABI_FUNC_VIS _Unwind_Reason_Code
+__gxx_personality_v0(int version,
```

Comment:
Can we put this function in `cxa_exception_emscripten.cpp`? That's a file we use for emscripten-specific stuff, and one stub function may not warrant a separate file.

Also nit: we don't usually use more than one blank lines between functions/headers